stdlib/2and3/pdb.pyi

```diff
@@ -19,3 +20,41 @@ else:
 
 def post_mortem(t: Optional[Any] = ...) -> None: ...
 def pm() -> None: ...
+
+class Pdb(Cmd):
+    if sys.version_info >= (3, 6):
+        def __init__(
+            self,
+            completekey: str = ...,
+            stdin: Optional[IO[str]] = ...,
+            stdout: Optional[IO[str]] = ...,
+            skip: Iterable[str] = ...,
+            nosigint: bool = ...,
+            readrc: bool = ...,
+        ) -> None: ...
+    elif sys.version_info >= (3, 2):
+        def __init__(
+            self,
+            completekey: str = ...,
+            stdin: Optional[IO[str]] = ...,
+            stdout: Optional[IO[str]] = ...,
+            skip: Iterable[str] = ...,
+            nosigint: bool = ...,
+        ) -> None: ...
+    else:
+        def __init__(
+            self,
+            completekey: str = ...,
+            stdin: Optional[IO[str]] = ...,
+            stdout: Optional[IO[str]] = ...,
+            skip: Iterable[str] = ...,
+        ) -> None: ...
+    def run(self, statement: str, globals: Optional[Dict[str, Any]] = ...,
+            locals: Optional[Dict[str, Any]] = ...) -> None: ...
+    def runeval(self, expression: str, globals: Optional[Dict[str, Any]] = ...,
+                locals: Optional[Dict[str, Any]] = ...) -> Any: ...
+    def runcall(self, *args: Any, **kwds: Any) -> Any: ...
```

Comment:
First argument `func` is missing. Return value of that callable is returned from `runcall()`.